### ISSUE TITLE
Backspace

### ISSUE THREAD

**Issue by mrquincle** [jessevdk/cldoc#84]
This issue I post with some hesitation. In our documentation the `backspace` doesn't work consistently. You can check the behavior at http://dobots.github.io/bluenet/. So, after clicking on a class, and then the `backspace` the URL is updated to http://dobots.github.io/bluenet/, but the page itself doesn't get refreshed.

However, I'm not entirely certain if this the case with every website generated by `cldoc`...

**Comment by jessevdk** [comment]
Yes, I have noticed this issue as well. It should be a general issue with cldoc.

**Comment by cbiffle** [comment]
I notice this specifically when the navigation was from the no-fragment case to the with-fragment case.  Example:

- `http://localhost:6060/`
- `http://localhost:6060/#etl/etl`
- Back.
- URL bar reads `http://localhost:6060/`
- Whatever you're using to be notified of fragment changes doesn't apparently get called.

**Comment by noirb** [comment]
I think this could be handled in the [popstate event](https://github.com/jessevdk/cldoc/blob/master/html/coffee/page.coffee#L450-L460). If `e.originalEvent.state` is undefined, it should be safe to load the main index page, since no state is ever pushed for that page.

I'm also seeing what looks like a related issue which I can't figure out, but it only affects sites I build myself, not the [online example](https://jessevdk.github.io/cldoc/example/) so it may be a configuration issue on my end (but it does affect the example site if I build it myself): if I navigate through two or more links, when I hit the Back button in my browser, the most recent page is skipped, and a page from earlier in the history is loaded (although the URL bar displays the correct page).
